### PR TITLE
refactor: update streetname registry

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/Public.Api/Feeds/Streetnames.cs [hunk 1]
```diff
@@ -14,8 +11,11 @@ namespace Public.Api.Feeds
     using Microsoft.AspNetCore.Mvc.Infrastructure;
     using Microsoft.Extensions.Options;
     using RestSharp;
-    using StreetNameRegistry.Api.Legacy.StreetName.Responses;
     using Swashbuckle.AspNetCore.Filters;
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using StreetNameRegistry.Api.Legacy.StreetName.Sync;
```
Comment (ArneD) [inline src/Public.Api/Feeds/Streetnames.cs:18]: Reorder usings

File: src/Public.Api/Infrastructure/Configuration/StreetNameOptionsV2.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 namespace Public.Api.Infrastructure.Configuration
 {
-   using StreetNameRegistry.Api.Oslo.Infrastructure.Options;
+    using StreetNameRegistry.Api.Oslo.Abstractions.Infrastructure.Options;
```
Comment (ArneD) [inline src/Public.Api/Infrastructure/Configuration/StreetNameOptionsV2.cs:3]: whitespace

File: src/Public.Api/StreetName/Oslo/StreetNameOsloController-Count.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@ namespace Public.Api.StreetName.Oslo
     using Microsoft.AspNetCore.Mvc.Infrastructure;
     using RestSharp;
     using StreetNameRegistry.Api.Legacy.StreetName.Query;
-    using StreetNameRegistry.Api.Legacy.StreetName.Responses;
+    using StreetNameRegistry.Api.Oslo.StreetName.Count;
```
Comment (jvandaal) [inline src/Public.Api/StreetName/Oslo/StreetNameOsloController-Count.cs:16]: Is this the right using import?

**Review by ArneD — APPROVED** [review]

**Pull request merged**